Cargo.toml

```diff
@@ -9,7 +9,7 @@ keywords = ["deku", "bits", "serialization", "deserialization", "struct"]
 categories = ["encoding", "parsing", "no-std"]
 description = "bit level serialization/deserialization proc-macro for structs"
 readme = "README.md"
-rust-version = "1.67.1"
+rust-version = "1.71"
```

Comment:
This is also noted in the README. And I would expect a note in the CHANGELOG.

Reply:
done in https://github.com/sharksforarms/deku/pull/438/commits/c2e2de945949bd54d6d9e06f62984195de8df162